x-pack/plugins/endpoint/public/applications/endpoint/store/policy_details/index.test.ts

```diff
@@ -72,7 +72,8 @@ describe('policy details: ', () => {
     });
 
     it('windows process eventing is enabled', async () => {
-      expect(windowsEventing(getState())!.process).toEqual(true);
+      const config = policyConfig(getState());
+      expect(config!.windows.events.process).toEqual(true);
     });
```

Comment:
@parkiino - Looks good! Will we also be adding a test here for the network eventing for Windows. This PR says mac eventing, are we also going to add tests for Mac here? Or did you want me to execute those manually for now until we can automate them?

Reply:
i can add a test for mac as well